Makefile

```diff
@@ -34,7 +34,7 @@ test:
 
 # ci is a convenience target for CI builds.
 .PHONY: ci
-ci: local test
+ci: verify-modules local test container
```

Comment:
This will always build `velero/velero-plugin-example:master`.
We might want to use a different tag for this image built in CI.

Reply:
we're not pushing it, so I don't think it really matters?

Reply:
that's fair. I am good w/ merging this